django_feature_flip/helpers.py

```diff
@@ -46,6 +48,13 @@ def feature_enabled_for_a_group_of_actor(self, feature, actor):
         groups_names = [group.name for group in feature.group_set.all()]
```

Comment:
```suggestion
        groups_names = group.name for group in feature.group_set.all()
```
This expression returns an iterator. do you really need to evaluate the whole list?